de-lib/src/main/java/org/iplantc/de/diskResource/client/presenters/DiskResourcePresenterImpl.java

```diff
@@ -539,20 +534,6 @@ public void onSendToEnsemblSelected(SendToEnsemblSelected event) {
         }
     }
 
-    @Override
-    public void onSendToTreeViewerSelected(SendToTreeViewerSelected event) {
-        // There is typically only one resource used.
-        for (DiskResource resource : event.getResourcesToSend()) {
-            InfoType infoType = InfoType.fromTypeString(resource.getInfoType());
-            if (infoType == null || !diskResourceUtil.isTreeInfoType(infoType)) {
-
-                announcer.schedule(new ErrorAnnouncementConfig(appearance.unsupportedTreeInfoType()));
-                return;
-            }
-            eventBus.fireEvent(new RequestSendToTreeViewerEvent((File)resource));
```

Comment:
I don't think I saw `RequestSendToTreeViewerEvent.java` removed, but I would think we could delete that (and any instances where a class implemented `onRequestSendToTreeViewer`)?

Same for `SendToTreeViewerSelected.java`.